airdrop/domain/models/airdrop_window.py

```diff
@@ -34,6 +35,7 @@ def to_dict(self):
         return {
             "airdrop_id": self._airdrop_id,
             "airdrop_window_id": self._window_id,
```

Comment:
do we already `_window_id` ? I don't see that time in the db though.

Reply:
we have airdrop_window_id already . and its the primary column